Terminal.Gui/Windows/Wizard.cs

```diff
@@ -90,6 +90,7 @@ public WizardStep (ustring title)
 				helpTextView.Y = 0;
 				helpTextView.ReadOnly = true;
 				helpTextView.WordWrap = true;
+				helpTextView.TabStop = false;
```

Comment:
I don't want to do this. I believe the way the most recent PR works is correct.

The user should be able to tab to the helpText so that scrolling can be done without the use of a mouse.

Reply:
You are right.